apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -395,6 +419,9 @@ k8s-celery-workers:
 		KUMA_CPU_REQUEST=${CELERY_WORKERS_CPU_REQUEST} \
 		KUMA_MEMORY_REQUEST=${CELERY_WORKERS_MEMORY_REQUEST} \
 		KUMA_ALLOWED_HOSTS="" \
+		NEW_RELIC_APP_NAME=${NEW_RELIC_CELERY_NAME} \
+		NEW_RELIC_MONITOR_MODE=${NEW_RELIC_MONITOR_MODE} \
+		NEW_RELIC_CONFIG_FILE=${NEW_RELIC_CONFIG_FILE} \
```

Comment:
I think these two lines:
```
NEW_RELIC_MONITOR_MODE=${NEW_RELIC_MONITOR_MODE} \
NEW_RELIC_CONFIG_FILE=${NEW_RELIC_CONFIG_FILE} \
```
are not needed since they will be picked up from the existing exported env variables?